SpywareFilter/sections/tracking_servers.txt

```diff
@@ -3316,6 +3316,8 @@
 ||signup-way.com^$third-party
 ||silverpop.com^$third-party
 ||silverpush.co^$third-party
+||simpleanalyticscdn.com^$third-party
```

Comment:
Hello @catleeball
We already have this rule in tracking_servers.txt

Reply:
It remains to be seen if someone uses that first domain in the CNAME to bypass the `$third-party`.